stravalib/model.py

```diff
@@ -533,9 +803,17 @@ class ActivityLap(
 
 
 class Map(PolylineMap):
+    """Pass through object. Inherits from PolyLineMap"""
+
     pass
 
 
+# if TYPE_CHECKING:
+#     from strava_model import Split
+
+
+# TODO - Split is already defined in strava_model - Name "Split" already
+# defined (possibly by an import) - i'm not sure how to deal with this.
```

Comment:
Ok here is another thing that i'd like to IGNORE for now but then open an issue about a better approach. 

if you look at this class `Split()`
it inherits from Split in the strava_model.py module.

it seems to be appending a few undocumented attrs and add a helper conversation attr.

i wondered if there might be an alternative way of doing this with composition? or creating a small class here called - SplitUndocumented and allowing the Split() class to inherit from that?

Thoughts? again i can open an issue but i'm going to ignore for now unless there is something here i don't understand.

Reply:
Without having tested it with mypy, could this naming clash not easily be resolved by defining the class as

```
class Split(strava_model.Split, BackwardCompatibilityMixin, DeprecatedSerializableMixin):
    ...
```
and importing `strava_model`?

Reply:
Yeah that should work. I've run into this issue a couple of time with classes that have the same name in models ans strava_model